src/Components/SideMenu/SideMenu.scss

```diff
@@ -0,0 +1,119 @@
+$sideBarWidth: 250px;
```

Comment:
Valdas, would you like to replace  ariables into main variables.scss file, just to store all variables at the same place?

Reply:
Done.